Organize assets directory, include README file (#204)
ewasm/design: assets/README.md

# Assets

> This directory contains some assets related to the ewasm project, such as

Reviewer: ```suggestion
This directory contains some assets related to the Ewasm project, such as
```

demonstration pictures and the logo design files and renders.

## Content

> - `README.md` - This file

Reviewer: ```suggestion
```

> - `blocks.png` - Example of blockin WebAssembly Text Format

Reviewer: ```suggestion
- `blocks.png` - Example of block in WebAssembly Text Format
```

- `if.png` - Example of counting wasm opcodes in an If statement.

> - `hexagon-sticker` - Files used to create the hexagon sticker4s with the Ewasm

Reviewer: ```suggestion
- `hexagon-sticker` - Files used to create the hexagon stickers with the Ewasm
```

  logo
- `logo` - Ewasm logo rendered files
- `logo-and-text` - Ewasm logo files which includes the text "ewasm"
- `src` - Source files (SVG, AI, EPG) corresponding to the logo.

> ### 

Reviewer: ```suggestion
```
